src/views/Orders.vue

```diff
@@ -82,7 +82,10 @@
             </ion-card-header>
             <ion-item>
               <ion-label class="ion-text-wrap">{{ $t("Days") }}</ion-label>
-              <ion-input :placeholder="$t('before auto cancelation')" />
+              <ion-input :placeholder="$t('before auto cancelation')" v-model.number="autoCancelDays" type="number" />
```

Comment:
I think we should also repopulate the already set value

Reply:
updated sir